.changeset/short-toes-relax.md

```diff
@@ -0,0 +1,10 @@
+---
+"@graphql-codegen/cli": minor
+"@graphql-codegen/core": minor
+"@graphql-codegen/plugin-helpers": minor
+"@graphql-codegen/client-preset": minor
+"@graphql-codegen/gql-tag-operations-preset": minor
+"@graphql-codegen/graphql-modules-preset": minor
+---
+
+Add a feature to transform documents
```

Comment:
It would be great if you could add an usage example here and also a section in the codegen documentation. 😇 The documentation change can be a follow-up PR.